src/Tools/dotnet-monitor/Extensibility/ProgramExtension.cs

```diff
@@ -21,18 +21,26 @@ internal partial class ProgramExtension : IExtension, IEgressExtension
         private readonly string _extensionName;
         private readonly string _targetFolder;
         private readonly string _declarationPath;
+        private readonly string _exePath;
         private readonly IFileProvider _fileSystem;
         private readonly ILogger<ProgramExtension> _logger;
-        private Lazy<ExtensionDeclaration> _extensionDeclaration;
+        public Lazy<ExtensionDeclaration> ExtensionDeclaration { get; }
```

Comment:
Why is this public? There's already a private property that you can make public instead.